sonatype/src/main/scala/org/typelevel/sbt/TypelevelSonatypePlugin.scala

```diff
@@ -50,31 +37,24 @@ object TypelevelSonatypePlugin extends AutoPlugin {
         "project /",
         "+mimaReportBinaryIssues",
         "+publish",
-        "tlSonatypeBundleReleaseIfRelevant")
+        "sonaRelease")
     }
   )
 
-  @nowarn("cat=deprecation")
   override def buildSettings =
     Seq(
-      tlSonatypeUseLegacyHost := false,
-      autoAPIMappings := true,
-      sonatypeCredentialHost := {
-        Option(System.getenv("SONATYPE_CREDENTIAL_HOST")).filter(_.nonEmpty).getOrElse {
-          if (tlSonatypeUseLegacyHost.value)
-            Sonatype.sonatypeLegacy
-          else
-            Sonatype.sonatype01
-        }
-      }
+      autoAPIMappings := true
     )
 
   override def projectSettings = Seq(
     publishMavenStyle := true, // we want to do this unconditionally, even if publishing a plugin
-    sonatypeProfileName := organization.value,
-    publishTo := sonatypePublishToBundle.value,
-    commands += sonatypeBundleReleaseIfRelevant,
-    apiURL := apiURL.value.orElse(hostedApiUrl.value)
+    publishTo := {
+      val centralSnapshots = "https://central.sonatype.com/repository/maven-snapshots/"
+      if (isSnapshot.value) Some("central-snapshots" at centralSnapshots)
+      else localStaging.value
+    },
+    apiURL := apiURL.value.orElse(hostedApiUrl.value),
+    sbtPluginPublishLegacyMavenStyle := false
```

Comment:
Without this, I get a 400 publishing to my own org.  I ran into this in sbt-http4s-org, too.

Reply:
Yes this seems to be an issue and this settings is sort of required. See:
* https://github.com/sbt/sbt/issues/8137
* https://github.com/sbt/sbt/issues/8133